### PR TITLE
add first two paras of subscription test article

### PR DESCRIPTION
**Pull request by CorneliaKelinske**
Thank you!

### CODE REVIEW

File: posts/2022/10-19-graphql_subscription_testing_pt1.md [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+│   │   ├── data_case.ex
+│   │   ├── fixtures
+│   │   │   └── user_fixtures.ex
+│   │   └── subscription_case.ex
+│   └── test_helper.exs
+```
+
+We see that, as per usual, the structure of our `test` folder mirrors the `lib` folder (shown in the previous post). 
+
+In the `support` folder, we want to make sure that `@endpoint` in `channel_case.ex` is set correctly (it should be by default).
```
Comment (alanvardy) [inline posts/2022/10-19-graphql_subscription_testing_pt1.md:53]: Are you going to provide an example of it being set correctly?

File: posts/2022/10-19-graphql_subscription_testing_pt1.md [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+                   }
+                 }
+               }
+             } = data
+    end
+  end
+end
+```
+
+As we can see in the example above, we start by building 'docs' both for the mutation we are subscribed to and for the corresponding subscription. When we write these docs, we are basically building the same query that our GraphiQL interface would build based on our input. We are specifying the arguments and types in the first line of the doc. In the second line, we have the name of our mutation or subscription and the arguments again, and below that is what the mutation/subscription returns.
```
Comment (alanvardy) [inline posts/2022/10-19-graphql_subscription_testing_pt1.md]:
```suggestion
As we can see in the example above, we start by building 'docs' (documents) both for the mutation we are subscribed to and for the corresponding subscription. When we write these docs, we are building the same query that our GraphiQL interface would build based on our input. We are specifying the arguments and types in the first line of the doc. In the second line, we have the name of our mutation or subscription and the arguments again, and below that is what the mutation/subscription returns.
```

File: posts/2022/10-19-graphql_subscription_testing_pt1.md [hunk 3]
```diff
@@ -0,0 +1,175 @@
...
+
+By passing the docs into `push_doc/3` later on in the subscription test (or, likewise, when we pass a doc into `Absinthe.run/3` in a mutation or query test), we utilize the internal API that Absinthe uses, thus bypassing the router.
```
Comment (alanvardy) [inline posts/2022/10-19-graphql_subscription_testing_pt1.md]:
```suggestion
By passing the docs into `push_doc/3` later on in the subscription test (or, likewise, when we pass a doc into `Absinthe.run/3` in a mutation or query test), we utilize the internal API that Absinthe uses, thus bypassing the router and increasing test performance.
```

**Review by alanvardy — APPROVED** [review]

File: posts/2022/10-19-graphql_subscription_testing_pt1.md (hunk 1 above)
Comment (CorneliaKelinske) [reply posts/2022/10-19-graphql_subscription_testing_pt1.md:53]: I am now :)

File: posts/2022/10-19-graphql_subscription_testing_pt1.md (hunk 3 above)
Comment (CorneliaKelinske) [reply posts/2022/10-19-graphql_subscription_testing_pt1.md]: Thank you!

**Pull request merged**